ui/app/AppLayouts/Profile/stores/ProfileSectionStore.qml

```diff
@@ -154,6 +158,51 @@ QtObject {
         }
     }
 
+    readonly property alias ownShowcaseCommunitiesModel: ownShowcaseModels.adaptedCommunitiesSourceModel
+    readonly property alias ownShowcaseAccountsModel: ownShowcaseModels.adaptedAccountsSourceModel
+    readonly property alias ownShowcaseCollectiblesModel: ownShowcaseModels.adaptedCollectiblesSourceModel
+    readonly property alias ownShowcaseSocialLinksModel: ownShowcaseModels.adaptedSocialLinksSourceModel
+
+    readonly property alias contactShowcaseCommunitiesModel: contactShowcaseModels.adaptedCommunitiesSourceModel
+    readonly property alias contactShowcaseAccountsModel: contactShowcaseModels.adaptedAccountsSourceModel
+    readonly property alias contactShowcaseCollectiblesModel: contactShowcaseModels.adaptedCollectiblesSourceModel
+    readonly property alias contactShowcaseSocialLinksModel: contactShowcaseModels.adaptedSocialLinksSourceModel
+
+    function requestContactShowcase(address) {
+        root.contactsStore.requestProfileShowcase(address)
+    }
+
+    function requestOwnShowcase() {
+        root.profileStore.requestProfileShowcasePreferences()
+    }
+
+    readonly property QObject d: QObject {
+        ProfileShowcaseSettingsModelAdapter {
+            id: ownShowcaseModels
+            communitiesSourceModel: root.communitiesList
+            communitiesShowcaseModel: root.profileStore.showcasePreferencesCommunitiesModel
+            accountsSourceModel: root.walletStore.ownAccounts
+            accountsShowcaseModel: root.profileStore.showcasePreferencesAccountsModel
+            collectiblesSourceModel: root.walletStore.collectibles
+            collectiblesShowcaseModel: root.profileStore.showcasePreferencesCollectiblesModel
+            socialLinksSourceModel: root.profileStore.showcasePreferencesSocialLinksModel
+        }
+
+        ProfileShowcaseModelAdapter {
+            id: contactShowcaseModels
```

Comment:
And here same thought than in the nim code.. for me it's confusing the `contact` naming when these models are not only managing `contacts` data but `general user's profile`. User can be a `contact` OR not! :)